src/helpers/vyos-vrrp-mdns.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+#
+# Copyright (C) 2021 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+case "$3" in
+    "MASTER"|"BACKUP"|"FAULT")
+        if [ -f /run/mdns_vrrp_active ]; then
```

Comment:
There are already VRRP/keepalived state transition scripts but I'm not into detail. Maybe @DmitriyEshenko knows more?

Reply:
This handler is called by the existing transition script: https://github.com/vyos/vyos-1x/pull/887/commits/3a77a1f25aabd92a1785fcdc8c89d65a2cf8d5de#diff-33fc5560ca57968dd54276c71f0a4c5e29f2ce34ff4d937b58672e112a19c75aR127